dashboard/config/scripts/levels/Dance_Party_11b.level

```diff
@@ -138,7 +140,7 @@
           <title name="PROPERTY">"scale"</title>
         </block>
         <block type="Dancelab_setTintInline">
-          <title name="SPRITE">dancer1</title>
+          <title name="SPRITE">new_dancer</title>
```

Comment:
I see `left_dancer` and `right_dancer` in this level, not `new_dancer`.  Are we okay with that for the toolbox?

Reply:
I've kept all the dancer names as they were in prod before the new tint block was added. We had `new_dancer` for all the toolbox blocks on this level